lib/cartopy/tests/test_polygon.py

```diff
@@ -192,8 +192,8 @@ def test_tiny_point_between_boundary_points(self):
         area = projected.area
         # Before fixing, this geometry used to fill the whole disk. Approx
         # 1.2e14.
-        self.assertTrue(2.4e9 < area < 2.5e9,
-                        msg='Got area {}, expecting ~2.4e9'.format(area))
+        self.assertTrue(81330 < area < 81340,
+                        msg='Got area {}, expecting ~81336'.format(area))
```

Comment:
It was surprising to me too. I've visually checked it, and it all looks good. I guess the important point is that the full disk is 10 orders of magnitude larger than area, where before it was 5 orders of magnitude. Given that this geometry was right on the edge, the act of reducing the edge is reasonably going to have a reasonable impact on the overall area.